server/app.js

```diff
@@ -42,10 +44,13 @@ app.use("/users", auth, usersRouter);
 app.use("/login", loginRouter);
 app.use("/spotify", spotifyRouter);
 app.use("/posts", auth, postsRouter);
-app.use("/follow", followRouter);
```

Comment:
i think this should not be removed

Reply:
It wasn't.  It was just moved lower.  I misunderstood, removed it, and have now reinserted it.